samples/bluetooth/hci_rpmsg/prj.conf

```diff
@@ -10,9 +8,4 @@ CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=512
 CONFIG_BT=y
 CONFIG_BT_HCI_RAW=y
 CONFIG_BT_MAX_CONN=16
-CONFIG_BT_CTLR_ASSERT_HANDLER=y
```

Comment:
This is required to generate HCI Hardware Error Event.

Reply:
In hci_rpmsg sample, the application-defined handler actually performs BT_ASSERT_MSG. In the default configuration logger is disabled, so I think it actually leads to the same behavior as for "default" handling mechanism. Did I overlook something ?
I noticed dedicated event in hci_uart sample.

Reply:
Hi @cvinayak, could you look at this one again ?

Reply:
As the title of this PR wants to have release configuration, the `CONFIG_BT_CTLR_ASSERT_HANDLER=y`  should not be removed. Instead, the assert should be encoded as "HCI Hardware Error event" and sent to application core. hci_uart sample on application core plus the hci_rpmsg sample on network core will be used for conformance testing and it is needed to capture asserts, if any, in the HCI trace in the conformance tester.

Adding the HCI Hardware Error event is outside the scope of this PR, but `CONFIG_BT_CTLR_ASSERT_HANDLER` should not be removed, as `CONFIG_BT_CTLR_ASSERT_HANDLER` is a release configuration for hci_rpmsg sample.

Reply:
Added `CONFIG_BT_CTLR_ASSERT_HANDLER` back to default configuration. Thank you for the explanation.